sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/implementation/jms/AzureServiceBusJmsPropertiesEnvironmentPostProcessorTests.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.autoconfigure.implementation.jms;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.mock.env.MockEnvironment;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class AzureServiceBusJmsPropertiesEnvironmentPostProcessorTests {
+
+    private final AzureServiceBusJmsPropertiesEnvironmentPostProcessor processor =
+        new AzureServiceBusJmsPropertiesEnvironmentPostProcessor();
+    private final MockEnvironment environment = new MockEnvironment();
+
+    @Test
+    void testDefaultPool() {
```

Comment:
let's add more test cases here:
- If users set the cache, true or false, then no pool env set
- If users set the pool, no changes made
- If users set both cache and pool, then no changes made